src/controllers/guests/posts.rs

```diff
@@ -109,9 +69,6 @@ pub async fn show_posts(
 ) -> Result<HttpResponse, actix_web::Error> {
     let db = &config.database_connection;
     let titles = path.parse::<i32>().unwrap_or_default();
-    let single_post = query_single_post(titles, db)
-        .await
-        .map_err(actix_web::error::ErrorInternalServerError)?;
 
     let single_post_struct = single_post_db(titles, db)
```

Comment:
Rename it as `single_post` or just `post`

Reply:
Fixed it